packages/nodes-base/nodes/ThrowError.node.json

```
@@ -0,0 +1,25 @@
+{
+	"node": "n8n-nodes-base.throw",
```

depending on name decision:
```suggestion
	"node": "n8n-nodes-base.throwError",
```